onadata/apps/logger/models/instance.py

```diff
@@ -388,6 +385,12 @@ def get_full_dict(self, load_existing=True):
             if not self.date_created:
                 self.date_created = submission_time()
 
+            if not self.date_modified:
+                doc[DATE_MODIFIED] = ""
```

Comment:
Could we simply not include the field or simply use the `date_created` value, since that would ideally be the same value on the initial creation of the record.